java/src/jmri/jmrit/logixng/Table.java

```diff
@@ -136,7 +138,39 @@ default public void setCell(Object value, String row, String column)
      * @throws ColumnNotFoundException if the column is not found
      */
     public int getColumnNumber(String columnName) throws ColumnNotFoundException;
-    
+
+    /**
+     * The available types of CSV from which to load a table
+     * The default is TABBED, as that was previously the only choice
+     * TABBED results in parsing the CSV file with tabs as the delimiters
+     * RFC (for RFC-4180) parses with commas as the delimiter, but dut does not allow empty lines
+     */
+    public enum CsvType {
+
+        TABBED, RFC;
```

Comment:
Thanks for moving this enum here.

Please take a second look at:
https://github.com/JMRI/JMRI/blob/master/java/src/jmri/jmrit/logixng/actions/IfThenElse.java#L544

But the enum isn't implemented correctly. Please use this code:

```
    public enum CsvType  {
        TABBED(Bundle.getMessage("CsvType_Tabbed")),
        COMMA(Bundle.getMessage("CsvType_Comma"));

        private final String _text;

        private CsvType(String text) {
            this._text = text;
        }

        @Override
        public String toString() {
            return _text;
        }

    }
```

I have changed TABBED to COMMA.